posts/2020-08-12-open-liberty-tools-intellij.adoc

```diff
@@ -0,0 +1,87 @@
+        <artifactId>liberty-maven-plugin</artifactId>
+        <version>3.2.2</version>
+    </plugin>
+----
+
+For Gradle projects, dev mode requires the Liberty Gradle Plugin. Specify the latest version of the Liberty Gradle Plugin in your project’s `build.gradle` file.
+[source,groovy]
+----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.0'
+    }
+}
+----
+
+== Interacting with dev mode
+
+Once the Liberty Maven Plugin is specified in your `pom.xml` file, or the Liberty Gradle Plugin is specified in your `build.gradle` file, your project name will be listed under the Liberty Dev Dashboard.  You can interact with dev mode by right-clicking on your project name, or selecting a command from the drop-down below the project name.  Alternatively, you can use the "Run Liberty Dev Task” option in the dashboard toolbar.
+
+[.img_border_light]
+image::/img/blog/olt-intellij-dashboard.gif[Selecting a task to run from the Liberty Dev Dashboard]
+
+== Commands
+
+The following commands can be run via the Liberty Dev Dashboard: 
+
+[cols="3,10",options="header"]
+|=========================================================
+|Command | Description
+| Run tests | Runs the unit tests and integration tests that are configured for your project. This command requires dev mode to be already started.
+| Start | Starts dev mode.
+| Start... | Starts dev mode with custom parameters.  For example, to run tests automatically after every change, include the `-DhotTests` parameter.  Additional supported parameters can be found in the documentation for the https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#additional-parameters[dev goal of the Liberty Maven Plugin] and https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#command-line-parameters[libertyDev task of the Liberty Gradle Plugin].
+| Stop | Stops dev mode.
+| View integration test report | Views the integration test report file.
+| View unit test report | Views the unit test report file.
+|=========================================================
+
+**Note:** Gradle projects only have a single "View test report" command.
+
+== Additional Resources
+
+For more information on Open Liberty dev mode, see our previous blog posts on Open Liberty dev mode with https://openliberty.io/blog/2019/10/22/liberty-dev-mode.html[Maven] and https://openliberty.io/blog/2020/03/11/gradle-dev-mode-open-liberty.html[Gradle].  Additionally, refer to the documentation for the https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev[dev goal of the Liberty Maven Plugin] or the  https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#libertydev-task[libertyDev task of the Liberty Gradle Plugin].
+
+If you prefer to develop with Visual Studio Code, check out our blog post on https://openliberty.io/blog/2019/11/13/liberty-dev-mode-vscode.html[Open Liberty dev mode with VS Code].
```

Comment:
Seems there are too many mentions of "blog post".  Perhaps this part can just be `check out https://openliberty.io/blog/2019/11/13/liberty-dev-mode-vscode.html[Open Liberty dev mode with VS Code].`